katello/katello/katello-backup-rotate-tar.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+## For this script it's advisable to use a shell, such as Bash,
+## that supports a TAR_FD value greater than 9.
+
+# echo Preparing volume $TAR_VOLUME of $TAR_ARCHIVE.
```

Comment:
is this line meant to be here?

Reply:
I just did a quick copy from the tar source, I'll get that fixed up